app/main/storage/jsonAesLevel.ts

```diff
@@ -23,7 +24,9 @@ const createAesSettings = (pbkdPassword: string): AesCipherSettings => ({
  * @param {string} id
  * @returns Promise<Storage<Buffer, JsonSerializable, Buffer, Buffer>>
  */
-export async function createJsonAesLevelStorage(args: any) {
+export async function createJsonAesLevelStorage(
+  args: any
+): Promise<Storage<Buffer, JsonSerializable, Buffer, Buffer>> {
```

Comment:
I kind of prefer leaving the "classic" signature of the function in a single line and moving the return type to the following line:
```ts
export async function createJsonAesLevelStorage(args: any)
  : Promise<Storage<Buffer, JsonSerializable, Buffer, Buffer>> {
```